packages/layout/src/components/__tests__/actuary.spec.tsx

```diff
@@ -12,10 +12,20 @@ import {
 	assertThatTitleWasSetToNullWhileFirstAndLastNamesWereLeftUnchanged,
 	clearTitleField,
 } from '../testHelpers/testHelpers';
-import { capitalizeEachWord } from '../../services';
```

Comment:
Is this used elsewhere? Otherwise I'd expect to see the function deleted too.

Reply:
it has been removed completely